ios/Classes/SwiftFlutterVideoCompressPlugin.swift

```diff
@@ -255,4 +262,36 @@ public class SwiftFlutterVideoCompressPlugin: NSObject, FlutterPlugin {
         exporter?.cancelExport()
         stopCommand = true
     }
+    
+    private func convertVideoToGif(_ path: String,_ startTime: NSNumber,_ endTime: NSNumber, _     duration:NSNumber,
+                           _ result: FlutterResult) {
+        let gifStartTime = Float(truncating: startTime)
+        var gifDuration = Float(truncating: 0)
+        
+        if gifStartTime > 0 {
+            if startTime.intValue > endTime.intValue {
+                result(FlutterError(code: channelName, message: "startTime should preceed endTime",
+                details: nil))
+            } else {
+                gifDuration  = Float(Float(truncating: endTime) - gifStartTime)
+            }
+        } else {
+            gifDuration = Float(truncating:     duration)
+        }
+        
+        let frameRate = Int(15)
```

Comment:
can this be configurable?

Reply:
It can, however, it complicates the plugin and I don't think it brings any value.